### PR TITLE
feat: ghost cms for blogs

### PR DESCRIPTION
**Pull request by yqrashawn**
use ghost CMS for blogs, docs will be added to notion

doc at https://www.notion.so/scrollzkp/CMS-Documentation-for-blog-3e06f0d20de84ac8b306fd65420be1e7?pvs=4

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206226769285544

### CODE REVIEW

**Comment by zzq0826** [comment]
image

The `og:image` seems use a wrong url

**Comment by Holybasil** [comment]
pls change base branch to "mainnet" @yqrashawn

File: src/pages/blog/detail.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const BlogDetail = () => {
       return anchor
     })
     try {
-      const blogPath = require(`../../assets/blog/${params.blogId.toLowerCase()}.md`)
+      const blogPath = `https://misc-pages-ghost-relay.vercel.app/api/post/${params.blogId.toLowerCase()}.md?title=1`
```
Comment (Holybasil) [inline src/pages/blog/detail.tsx:72]: is "?title=1" required?

Comment (yqrashawn) [reply src/pages/blog/detail.tsx:72]: yes, the markdown by default doesn't include the title

**Comment by yqrashawn** [comment]
og image fixed and PR now targting mainnet branch @zzq0826 @Holybasil

**Review by Holybasil — APPROVED** [review]

**Pull request merged**